pages/linux/logger.md

```diff
@@ -0,0 +1,23 @@
+# logger
+
+> Add messages to syslog.
```

Comment:
Actually `syslog` is the correct term because that's what it is. "system log" might mean that the logs are stored somewhere in the system. Should I add `/var/log/syslog` at the end ? Does that make it clearer ?